system/keyboardlayouts/chinese.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<!--
+Please use English language names instead.
+Default font lacks support for all characters
+-->
+<keyboardlayouts>
+  <layout language="Chinese" layout="BasePY" codingtable="BasePY">
```

Comment:
Different layout may use the same codingtable. e.g. <layout language="Chinese" layout="BasePY_QWERTY" codingtable="BasePY">